gems/logstash-core/CVE-2016-10362.yml

```diff
@@ -0,0 +1,20 @@
+---
+gem: logstash-core
+cve: 2016-10362
+ghsa: 3gg4-6hqg-2vjx
+url: https://web.archive.org/web/20210730201452/http://www.securityfocus.com/bid/99154
+title: Logstash Logs Sensitive Information
+date: 2016-06-16
+description: |
+  Prior to Logstash version 5.0.1, Elasticsearch Output plugin when updating
+  connections after sniffing, would log to file HTTP basic auth credentials.
+cvss_v2: 4.0
+cvss_v3: 6.5
+notes: Never patched
```

Comment:
The description says "Prior to Logstash version 5.0.1" and `logstash-core` is now on [version 7.5.2](https://rubygems.org/gems/logstash-core), so I think `patched_versions` should be `>= 5.0.1`.

Reply:
Made fix.